cmd/create/command.go

```diff
@@ -0,0 +1,73 @@
+package create
+
+import (
+	"io"
+	"os"
+
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/cobra"
+
+	"github.com/giantswarm/devctl/cmd/create/release"
+)
+
+const (
+	name        = "create"
+	description = "Create resources in the current working directory."
```

Comment:
Some examples would be neato here because the combination of flags can be quite complex from what I see.

Reply:
I added an example.